src/Xunit.NetCore.Extensions/Xunit.NetCore.Extensions.csproj

```diff
@@ -9,4 +9,7 @@
   <ItemGroup>
     <PackageReference Include="xunit.core" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\Shared\UnitTests\CustomXunitAttributesUtilities.cs" />
```

Comment:
Why does this need to be in Shared instead of made public in this assembly?

Reply:
No need for that, moved to `Xunit.NetCore.Extensions`.